### DISCUSSION TITLE
Generella frågor om appens funktion

### DISCUSSION THREAD

**Discussion by dhogborg** [dhogborg/off-peak#32] [Q&A]
Du kanske har frågor om appens funktion som inte direkt kan sägas passar som en Issue? 
Välkommen att ställa sådana frågor här.

**Comment by AndLin67** [comment]
Hej
Har solceller och Tibber. När jag kollar i appen så visar summeringen att jag "Förlorar" på timpris. Men graferna visar att jag ligger under medelspotpris i stort sett hela perioden och ser att majoriteten av förbrukning ligger på "billiga" tider.   Tror att solcellerna spökar i er beräkning!
Kan också nämna att jag styr all förbrukning mot pris med Homey. Har också en annan fastighet utan solceller dör jag styr på samma sätt. Där verkar appen visa rätt!

**Comment by dhogborg** [comment]
Hej!
Solcellerna borde inte påverka ditt resultat nämnvärt, särskilt inte under vinterförhållanden. 

Däremot så påverkar energiprisets variation under hela perioden ditt resultat. Om du förbrukar mycket el på dagar då elen är dyr kan ditt resultat för hela månaden skevas så att timpris blir en förlustaffär, trots att du varje enskild dag varit bättre än "snittet".

Det är så klart viktigt att flytta förbrukning mot billiga timmar, men även att flytta förbrukning till billiga dagar har stor effekt.

**Comment by AndLin67** [comment]
Hej! Något skumt är det. Detta är min förbruknings profil för januari. Mesta förbrukning ligger billiga timmar och appen visar -2% mot medeldpot.
Anders
/
![Screenshot_20240126-222139](https://github.com/dhogborg/off-peak/assets/130901031/2325a7b4-4b9e-41ee-b452-395208589877)
